src/components/Footer.tsx

```diff
@@ -0,0 +1,75 @@
+import classNames from 'classnames';
+import React from 'react';
+import { SortType } from '../types/SortType';
+
+interface Props {
+  active: number;
+  completed: number;
```

Comment:
please give more clear name then 'completed'...it is not clear from the name that it will be a number